src/cli/test/test_measure_manager.py

```diff
@@ -255,6 +332,8 @@ def test_download_bcl_measures(measure_manager_client: MeasureManagerClient, exp
     r = measure_manager_client.post(url="/download_bcl_measure", json={"uid": TEST_BCL_MEASURE_UID})
     r.raise_for_status()
     measure_data = r.json()
+    if measure_manager_client.is_classic:
+        measure_data = measure_data[0]
```

Comment:
This is the only real difference, in `/download_bcl_measure`

* Classic (Ruby) CLI uses to return a single-element list
* C++ CLI returns the element directly